### DISCUSSION TITLE
Exploration: Show Usage Stats

### DISCUSSION THREAD

**Discussion by Overload119** [notaapp/talk#222] [Feature Requests]
Tools that you use everyday (such as Nota) become more sticky when you realize how much you use them.

A lot of apps let you see simple stats such as how many words you write per day and the evolution of those notes over time.
I have a.git in my notes directory that allows me to see this for example. 
However, this could be baked into Nota directly - at first at a very small scale, such as # of words written per day. 

This is just an idea that can make users more sticky and continue to use Nota. 

(attached is Alfred showing something similar)

![image](https://user-images.githubusercontent.com/1012072/95031771-9e7a9380-066c-11eb-86ef-888b94cb81cf.png)

Also, Obsidian does this with their Graph View which people share on social media, generating buzz for their product.
In reality Graph View is not that useful outside of seeing a pretty visualization.

**Comment by astoilkov** [comment]
We are definitely considering stats. We even have an [item in our roadmap](https://github.com/notaapp/issues/issues/21) for that.

**Comment by astoilkov** [comment]
We started experimenting with this. We have a private experiment that collets usage data when we work and we started experimenting with visualizing interesting/useful information.
